lib/drivers/driver.js

```diff
@@ -163,4 +163,42 @@ Driver.prototype.push = function (opts) {};
  */
 Driver.prototype.status = function (opts) {};
 
+/**
+ * Track local files into a git repository. Performs a post request to /track
+ * endpoint with a JSON payload in the form of:
+ *
+ *
+ *          {
+ *            "message": "Commit message",
+ *            "files": [{
+ *                "name": "README.md",
+ *                "status": "modified"
```

Comment:
Ok, so we should mention that `status` is useful. And as long as there is no documentation for Uhub, it's good to have it written somewhere, like here.

```
Added -> File will be added and committed
Copied -> ?
Deleted -> File will be removed
Modified -> Modification will be committed
Renamed -> ?
Unmodified -> Nothing
Untracked -> File will be added and committed
```